Integrations/integration-CrowdStrikeFalcon.yml

```diff
@@ -0,0 +1,1095 @@
+            :return: Response json of the get detection entities endpoint (detection objects)
+        """
+        ids_json = {'ids': detections_ids}
+        if detections_ids:
+            response = http_request(
+                'POST',
+                '/detects/entities/summaries/GET/v1',
+                data=json.dumps(ids_json)
+            )
+            return response
+        return detections_ids
+
+
+    def create_ioc():
+        """
+            UNTESTED - Creates an IoC
+            :return: Response json of create IoC request
+        """
+        args = demisto.args()
+        input_args = {}
+        # req args:
+        input_args['type'] = args['ioc_type']
+        input_args['value'] = args['ioc_value']
+        input_args['policy'] = args['policy']
+        # opt args:
+        input_args['expiration_days'] = args.get('expiration_days')
+        input_args['source'] = args.get('source')
+        input_args['description'] = args.get('description')
+
+        payload = {k: str(v) for k, v in input_args.items() if v}
+        headers = {'Authorization': HEADERS['Authorization']}
+        return http_request('POST', '/indicators/entities/iocs/v1', params=payload, headers=headers)
+
+
+    def search_iocs():
+        """
+            UNTESTED IN OAUTH 2- Searches an IoC
+            :return: IoCs that were found in the search
+        """
+        args = demisto.args()
+        ids = args.get('ids')
+        if not ids:
+            search_args = {
+                'types': str(args.get('ioc_types', '')).split(','),
+                'values': str(args.get('ioc_values', '')).split(','),
+                'policies': str(args.get('policy', '')),
+                'sources': str(args.get('sources', '')).split(','),
+                'from.expiration_timestamp': str(args.get('expiration_from', '')),
+                'to.expiration_timestamp': str(args.get('expiration_to', '')),
+                'limit': str(args.get('limit', 50))
+            }
+            payload = {}
+            for k, arg in search_args.items():
+                if type(arg) is list:
+                    if arg[0]:
+                        payload[k] = arg
+                elif arg:
+                    payload[k] = arg
+            ids = http_request('GET', '/indicators/queries/iocs/v1', payload).get('resources')
+            if not ids:
+                return None
+        else:
+            ids = str(ids)
+        payload = {
+            'ids': ids
+        }
+        return http_request('GET', '/indicators/entities/iocs/v1', params=payload)
+
+
+    def enrich_ioc_dict_with_ids(ioc_dict):
+        """
+            Enriches the provided ioc_dict with IoC ID
+            :param ioc_dict: IoC dict transformed using the SEARCH_IOC_KEY_MAP
+            :return: ioc_dict with its ID key:value updated
+        """
+        for ioc in ioc_dict:
+            ioc['ID'] = '{0}:{1}'.format(ioc.get('Type'), ioc.get('Value'))
```

Comment:
please use named place holders when formatting more than one variable in a string.